src/range/padding.rs

```diff
@@ -24,9 +35,7 @@ pub struct RangeProofPadding {
 
 impl RangeProofPadding {
     pub fn get_aggregated(&self) -> &RangeProof {
-        if self.aggregated.is_empty() {
-            panic!(); // TODO
-        }
+        AggProofEmpty::new(&self.aggregated);
```

Comment:
This wasn't the code I was referring to, it was this code further down:
```rust
impl RangeProvable for RangeProofPadding {
    fn new(aggregated: &[RangeProof], individual: &[RangeProof]) -> Self {
        if aggregated.len() > 1 {
            panic!(); //TODO
        }
        RangeProofPadding {
            aggregated: aggregated.to_vec(),
            individual: individual.to_vec(),
        }
    }
...
```
But both bits of code need to be changed anyway. The idea is to actually change the type inside the struct, not just to call another function.